.travis.yml

```diff
@@ -1,5 +1,5 @@
 rvm:
-  - 2.0.0
+  - 2.0.0-p353
```

Comment:
i'm not sure greenfield use that patch level since they are using an apt package.

also, travis use an rvm option that tries to find a binary version of the ruby, if that patch level is not there.
it will take longer.

should I want to specify a patch level, I would put the latest one

my two cents